client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -29,11 +32,13 @@ import {
 } from '../../StockOut';
 import { DraftStockOutLine } from '../../types';
 import { isA } from '../../utils';
+import { AccordionPanelSection } from './PanelSection';
+import { PrescriptionLineEditTable } from './PrescriptionLineEditTable';
 
 interface PrescriptionLineEditFormProps {
-  allocatedQuantity: number;
-  availableQuantity: number;
-  item: DraftItem | null;
+  allocatedUnits: number;
+  availableUnits: number;
```

Comment:
Have tried to be explicit whether each variable is in Units or Packs -- let me know if I've missed any, or it's unclear.